<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dockPlantsDialog</class>
 <widget class="QDialog" name="dockPlantsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLANTS Docking Parameters</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_exec">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PLANTS Executable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SporesExec">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Folder containing python scripts to convert receptor and ligand files in PDBQT format.&lt;/p&gt;&lt;p&gt;Usually contained in 'MGLTools-{&lt;span style=&quot; font-style:italic;&quot;&gt;version&lt;/span&gt;}/MGLToolsPckgs/' directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_SporesExec">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>70</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Folder containing python scripts to convert receptor and ligand files in PDBQT format.&lt;/p&gt;&lt;p&gt;Usually contained in 'MGLTools-{&lt;span style=&quot; font-style:italic;&quot;&gt;version&lt;/span&gt;}/MGLToolsPckgs/' directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_cz">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_exec_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SPORES Executable</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_PlantsExec">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>40</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Docking with PLANTS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>20</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_cy">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>20</width>
     <height>148</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PlantsExec">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_cx">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Binding Site</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_radius">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_scoringFunction">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;- plp95: Piecewise Linear Potential from Gehlhaar DK et al&lt;/p&gt;&lt;p&gt;- plp: PLANTS version of the Piecewise Linear Potential&lt;/p&gt;&lt;p&gt;- chemplp: PLANTS version of the Piecewise Linear Potential implementing GOLD's terms&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>chemplp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>plp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>plp95</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Scoring Function</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_water">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>266</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_water">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Include structural water molecule</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_searchSpeed">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>138</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;- speed1&lt;/span&gt;: highest reliability but slowest setting&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;- speed4&lt;/span&gt;: 4 times faster than speed1 but modest reliability&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>speed1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>speed2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>speed4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Search speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>168</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of ants</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ants">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>164</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default: 20&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_evaporationRate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>196</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pheromone evaporation rate. Higher rate usually leads to faster convergency.&lt;/p&gt;&lt;p&gt;Default: 0,15&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>0.150000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaporation rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>232</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Iteration scaling factor</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_iterationScaling">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>228</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Multiplies the number of iterations done by the ACO algorithm. Used to tune the tradeoff between computation time and solution quality.&lt;/p&gt;&lt;p&gt;Default: 1,0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
